src/interfaces.py

```diff
@@ -533,7 +533,7 @@ class Input(np.ndarray):
 
     The ordering of registers:  RAX, RBX, RCX, RDX, RSI, RDI, FLAGS
     """
-    seed: int = 0
+    seed: int = 0  # deprecated?
```

Comment:
Agreed that this is deprecated. It looks like the only thing `class Input`'s `seed` field is used for is for string representations (`__str__()` and `__repr__()`). It could be removed and something else could be used for the two dunder methods.

Reply:
Yep. But I'll leave it for the time being - there might be some use cases I'm forgetting.